src/main/java/edu/kis/powp/jobs2d/features/canvasDrawArea/CanvasSize.java

```diff
@@ -0,0 +1,7 @@
+package edu.kis.powp.jobs2d.features.canvasDrawArea;
+
+import java.awt.*;
+
+public interface CanvasSize {
+    Dimension getDimension();
```

Comment:
Very limited definition of canvas, only rectangular shapes, no info about coordinates. But ok.